docs/releases/1.26-NOTES.md

```diff
@@ -6,6 +6,18 @@ This is a document to gather the release notes prior to the release.
 
 # Significant changes
 
+## AWS meta data service hop limit
+
+As of Kubernetes version 1.26 and with IRSA enabled, control plane nodes will now run with a max hop limit of 1 for the metadata service. This will prevent Pods without host networking from accessing the instance metadata service.
```

Comment:
Should be in the "AWS only" section

Reply:
AWS only section is under "other significant changes", but this requires more attention than that. Doesn't the title indicate that it is AWS only?

Reply:
Why does it require more attention than that?

Reply:
It's borderline breaking change if you e.g run `kops create` as part of e2e testing.

Reply:
To have an adverse impact, one would have to be running one's own workload on the control plane, with non-host-network, not using IRSA, requiring a permission granted to the control plane instance role, not running on worker nodes (because they wouldn't have access to the permission), and the cluster would have to have `useServiceAccountExternalPermissions` enabled.

Otherwise, it's just that the control plane nodes run a little more securely. I don't see this as meeting the bar for a top-line announcement.

Reply:
Fair enough. Slimmed and moved under AWS